scripts/debugcelery.sh

```diff
@@ -6,6 +6,6 @@ set -e
 
 STOP_SERVICE="(sudo service celeryd stop || /bin/true)"
 CHANGE_DIR="cd /opt/app/"
-RUN_CELERY="envdir /etc/mmw.d/env celery -A 'mmw.celery:app' worker --autoreload -l debug -n debug@%n"
+RUN_CELERY="envdir /etc/mmw.d/env celery -A 'mmw.celery:app' worker --purge -l debug -n debug@%n"
```

Comment:
`--autoreload` no longer exists as an option; if we want to keep `--purge`, it will discard all prior pending messages on restarting the celery workers.